Sources/TensorFlow/Operators/LinearAlgebra.swift

```diff
@@ -148,4 +184,26 @@ public extension Tensor where Scalar: TensorFlowFloatingPoint {
     func qrDecomposition(fullMatrices: Bool = false) -> (q: Tensor<Scalar>, r: Tensor<Scalar>) {
         _Raw.qr(self, fullMatrices: fullMatrices)
     }
+
+    /// Returns the singular value decompositions of one or more matrices.
+    ///
+    /// Computes the SVD of each inner matrix in the `input` tensor such that
+    /// `input[..., :, :] = u[..., :, :] * diag(s[..., :, :]) * transpose(v[..., :, :])`
+    ///
+    /// - Parameters:
+    ///   - computeUv: If `true` then left and right singular vectors will be computed and
+    ///     returned in `u` and `v`, respectively. Otherwise, only the singular values will be
+    ///     computed, which can be significantly faster.
+    ///   - fullMatrices:  If `true`, compute full-sized `u` and `v`. If `false` compute only the
+    ///     leading `min(shape[rank - 1], shape[rank - 2])` singular vectors. Ignored if
+    //      `computeUv` is `false`.
+    @inlinable
+    func svd(computeUv: Bool = true, fullMatrices: Bool = false
```

Comment:
Can you rename `computeUv` to `computeUV`, `uv` or something more readable? It might follow the language style, but it is slightly obscure, what is meant by `computeUv`.

Reply:
I think I took this variable name from the raw ops generated. I do agree that it sounds sort of vague. 

But would like @dan-zheng's idea on this

Reply:
I think `computeUV` is the better name, since `Uv` is more easily misinterpreted as one camelcased entity than `UV`. Could you please update the parameter name and doc comments?

Reply:
A semantic name like `computeLeftAndRightSingularVectors` may be more idiomatic than `computeUV`. I'm not sure what's the best semantic name, so using `computeUV` for now sounds good to me, to be consistent with [`numpy.svd`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.linalg.svd.html) and [`tf.linalg.svd`](https://www.tensorflow.org/api_docs/python/tf/linalg/svd).